dffml/util/cli/cmd.py

```diff
@@ -109,11 +109,9 @@ def add_subs(self, add_from: "CMD"):
         for i, field in enumerate(dataclasses.fields(add_from.CONFIG)):
             arg = mkarg(field)
             if isinstance(arg, Arg):
-                position = field.metadata.get("position", None)
-                if not "default" in arg and position is None:
-                    position = i
-                if isinstance(position, int):
-                    position_list[position] = (field.name, arg)
+                position = None
+                if not "default" in arg:
```

Comment:
I think this might be what we really want here, because a few of the arguments that you've just defaulted to `None` really are required. I'm sorry I believe I remember saying that we might not have any more cases like this in the meeting, however, `model` and `sources` seem to be those cases where we have a non positional argument that is required.

```suggestion
                if not "default" in arg and not arg.get("required", False):
```